File: testjavabenchmark/domain/shortcharacter/twitter/Account.java
package domain.shortcharacter.twitter;

import infrastructure.socialmedia.SocialNetwork;
//Functional requirement 3.1.1
//Test case 36: Class domain.shortcharacter.twitter.Account may only have a dependency with class infrastructure.socialmedia.SocialNetwork
//Result: TRUE
public class Account {	
	public Account(){
		//FR5.1
		new SocialNetwork();
	}
}

File: testprojects/c#/domain/shortcharacter/identica/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.asocialmedia;

namespace CSharpBenchmark.domain.shortcharacter.identica
{
    //Functional requirement 3.1.1
    //Test case 37: Class domain.shortcharacter.identica.Account may only have a dependency with class infrastructure.asocialmedia.ASocialNetwork
    //Result: FALSE
    public class Account
    {
        public Account()
        {
            //FR5.1
            new ASocialNetwork();
        }
    }
}